### PR TITLE
added login and signup page

### PR DESCRIPTION
**Pull request by anupkgurung**
added login and signup page.
added navbar and sidebar.
added input, footer and video card component.

### CODE REVIEW

File: src/pages/auth.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    width: 100%;
+    margin-top: 3rem;
+    padding: 1rem;
+}
+.login-form , .signup-form {
+    width: 30rem;
+    margin: 5rem;
+    padding: 3rem 3rem;
+    border-radius: 5px;
+    box-shadow: 1px 2px 5px black;
```
Comment (ranitasahaa1230) [inline src/pages/auth.css:16]: You can put the black in your root folder and use it as a color variable

File: src/pages/style.css [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+}
+.content-body-product {
+    display: grid;
+    grid-template-columns: 23% 77%;
+    grid-gap: 1rem;
+    margin-top: 6rem;
+    padding: 2rem 1rem;
+}
+.aside-container {
+    border-right: 1px solid #00000038;
```
Comment (ranitasahaa1230) [inline src/pages/style.css:18]: Same for this

File: src/pages/style.css [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+    display: flex;
+    align-items: center;
+    margin: 0 0.3rem 0rem 0.2rem;
+    padding: .75rem;
+    justify-content: space-evenly;
+    border-radius: 20px;
+    height: 2rem;
+}
+.chip {
+    background-color: #efefef;
```
Comment (ranitasahaa1230) [inline src/pages/style.css:161]: Same for this too

File: src/components/Input.jsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+                <div className="show-paswrd flex-row padding-top-2">
+                    <input type={type} id={id} className="checkbox" onChange={onChangeHandler} />
+                    <label htmlFor={id}>{title}</label>
+                </div>
+            )
+            break;
+
+        case "email":
+        case "text":
+        case "password":
```
Comment (ranitasahaa1230) [inline src/components/Input.jsx:22]: Nice way of using the `switch` inside the input component

File: src/pages/style.css [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+    padding: 0;
+}
+.card-img-container {
+    height: 12rem;
+}
+.card-desc {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    /* gap: 2rem; */
```
Comment (ranitasahaa1230) [inline src/pages/style.css:71]: Remove this if not needed

**Review by ranitasahaa1230 — APPROVED** [review]
Loved the breakdown of components
Good use of `switch`
Do add a screenshot or a GIF in your description
Rest looks good

File: src/components/Sidebar/Sidebar.jsx [hunk 6]
```diff
@@ -0,0 +1,20 @@
+import { NavLink } from "react-router-dom";
+import { tempDataSidebar }  from "../../tempData";
+
+export const Sidebar = () => {
+    return (
+        <>
```
Comment (SanketDhabarde) [inline src/components/Sidebar/Sidebar.jsx:6]: Since it has a single parent element `<aside>`. the fragment is not needed

**Review by SanketDhabarde — APPROVED** [review]
- Like the reusable `Input` component
- Like that, you are giving default prop 
- Looks good

**Pull request merged**